examples/showcase_1kt.rs

```diff
@@ -0,0 +1,240 @@
+//! An example showing Tock's capabilitities ported to CherryMotes, on 1KT.
+//! Every T seconds (a configurable constant) it infinitely sends a frame
+//! with a tuple (timestamp, CherryMote id, sequence number, measured temperature)
+//! and after each send receives N - 1 (a configurable constant) frames from other CherryMotes
+//! and prints them to console.
+
+#![no_main]
+#![no_std]
+#![forbid(unsafe_code)]
+
+use core::fmt::{Display, Write as _};
+use libtock::alarm::{Alarm, Milliseconds};
+// use libtock::console::Console;
+use libtock::console_lite::ConsoleLite;
+use libtock::ieee802154::{Ieee802154, RxBufferAlternatingOperator, RxOperator as _, RxRingBuffer};
+use libtock::runtime::{set_main, stack_size};
+use libtock::temperature::Temperature;
+
+set_main! {main}
+stack_size! {0xA00}
+
+/// Number of CherryMotes executing this app at the same time.
+const N_MOTES: usize = 2;
+
+/// Interval between frame transmissions.
+const N_SECS: u32 = 3;
+const SLEEP_TIME: Milliseconds = Milliseconds(1000 * N_SECS);
```

Comment:
Docelowo będzie to ustawiane przez UART-full?

Reply:
A nie wiem. Myślałem, że UART-full ma służyć miganiu diodką, a nie przekazywaniu parametrów działania.

Reply:
Można zrobić, że UART-full'em ustawia się parametry i jest to potwierdzane mignięciem LEDem – wtedy może byłoby to najmniej naciągane zastosowanie UART-full'a i LEDów :smile: